includes/wp-admin/class-settings.php

```diff
@@ -389,60 +389,152 @@ public static function add_settings_help_tab() {
+					'<p>' . \esc_html__( 'You can also configure global settings to control which post types (posts, pages, custom post types) are federated by default. This gives you both site-wide control and per-post flexibility to manage exactly how your content is shared.', 'activitypub' ) . '</p>' . "\n" .
+
+					'<h2>' . \esc_html__( 'Receiving Interactions', 'activitypub' ) . '</h2>' . "\n" .
+					'<p>' . \esc_html__( 'One of the most powerful features of the ActivityPub plugin is its ability to receive and display interactions from across the Fediverse. When someone on Mastodon or another Fediverse platform comments on your post, their comment appears directly in your WordPress comments section, creating a seamless conversation between your blog and the wider Fediverse.', 'activitypub' ) . '</p>' . "\n" .
+					'<p>' . \esc_html__( 'These Fediverse comments integrate naturally with your existing WordPress comment system. You can moderate them just like regular comments, and any replies you make are automatically federated back to the original commenter, maintaining the conversation thread across platforms.', 'activitypub' ) . '</p>' . "\n" .
+					'<p>' . \esc_html__( 'Beyond comments, the plugin also tracks likes and shares (boosts) from Fediverse users. These interactions can provide valuable feedback and help you understand how your content is being received across the decentralized social web.', 'activitypub' ) . '</p>' . "\n" .
+
+					'<h2>' . \esc_html__( 'Mentions and Replies', 'activitypub' ) . '</h2>' . "\n" .
+					'<p>' . \wp_kses( \__( 'The ActivityPub plugin enables true cross-platform conversations by supporting mentions and replies. When writing a post, you can mention any Fediverse user by using their full address format. For example, typing <code>@username@domain.com</code> will create a mention that notifies that user, regardless of which Fediverse platform they use.', 'activitypub' ), $code_html ) . '</p>' . "\n" .
+					'<p>' . \wp_kses( \__( 'Mentions use the format <code>@username@domain.com</code> and work just like mentions on other social platforms. The mentioned user receives a notification, and your post appears in their mentions timeline. This creates a direct connection between your WordPress site and users across the Fediverse.', 'activitypub' ), $code_html ) . '</p>' . "\n" .
+					'<p>' . \esc_html__( 'Similarly, when someone mentions your Fediverse identity in their post, you&#8217;ll receive an email notification that you can respond to with a new post. This two-way communication bridge makes your WordPress site a full participant in Fediverse conversations.', 'activitypub' ) . '</p>' . "\n",
 			)
 		);
 
+		// Editor Blocks.
+		\get_current_screen()->add_help_tab(
+			array(
+				'id'      => 'editor-blocks',
+				'title'   => \__( 'Editor Blocks', 'activitypub' ),
+				'content' =>
+					'<h2>' . \esc_html__( 'Introduction to ActivityPub Blocks', 'activitypub' ) . '</h2>' . "\n" .
```

Comment:
what about mentioning what to type in the block editor to see the block or is this self explaining?